Lesson8-Internet/app/src/main/java/com/example/android/marsrealestate/ui/screen/detail/DetailFragment.kt

```diff
@@ -14,7 +14,7 @@
  *  limitations under the License.
  */
 
-package com.example.android.marsrealestate.detail
+package com.example.android.marsrealestate.ui.screen.detail
```

Comment:
ui hoặc screen chứ không lồng 2 package nhé chú
```suggestion
package com.example.android.marsrealestate.screen.detail
```